tests/ui/conftest.py

```diff
@@ -1,20 +1,72 @@
 import datetime
 import json
 import os
+import random
 import urlparse
 
 import jwt
 import pytest
 import requests
 from django.core.management import call_command
 from fxapom.fxapom import DEV_URL, PROD_URL, FxATestAccount
-from olympia.amo.tests import create_switch
+from olympia import amo
+from olympia.addons.forms import icons
+from olympia.addons.models import AddonUser, Preview
+from olympia.addons.utils import generate_addon_guid
+from olympia.amo.tests import (
+    addon_factory,
+    create_switch,
+    user_factory,
+    version_factory,
+)
+from olympia.constants.applications import APPS, FIREFOX
+from olympia.constants.base import (
+    ADDON_EXTENSION,
+    ADDON_PERSONA,
+    STATUS_PUBLIC
+)
+from olympia.landfill.collection import generate_collection
+from olympia.landfill.generators import generate_themes
+from olympia.reviews.models import Review
+from olympia.users.models import UserProfile
 from pytest_django import live_server_helper
 
 
+@pytest.fixture
+def load_fixtures(request, my_base_url):
+    """
+    This fixture checks the base url. If it determines that a live server is
+    running, it will load the fixtures provided from the text.
+
+    If it determines that a live server is not running, it will load no
+    fixtures.
+
+    :returns: pytest fixture
+    """
+
+    # Checks if localhost is set in basegit url env variable
+    if 'localhost' in my_base_url:
+        # checks if fixture requested is a string or a list
+        if not isinstance(request.param, basestring):
+            # loops through list of fixtures provided from test
+            for fixture in request.param:
+                return request.getfuncargvalue(fixture)
+        else:
+            return request.getfuncargvalue(request.param)
```

Comment:
This was renamed to [getfixturevalue](https://docs.pytest.org/en/latest/builtin.html#_pytest.fixtures.FixtureRequest.getfixturevalue) in pytest 3, and should be avoided unless it's absolutely necessary. Perhaps there's no way around using this, but I'm not sure about looping through the arguments like this. I'd prefer each fixture was more explicit in the fixtures it depends on.